community-roadmap/backend/README.md

```diff
@@ -0,0 +1,60 @@
+# Community Roadmap for Zendesk Help Center
+
+This project contains a service that maps Zendesk community posts to product roadmap items. The service is built as a Cloudflare Worker with a Cloudflare KV data store.
+
+## Development
+
+The service can be run locally using Cloudflare's `wrangler` tool.
+
+Installation:
+```
+brew install wrangler
+```
+
+Run:
+```
+wrangler dev --env dev
+```
+or just:
+```
+npm start
+```
+
+## Test
+
+Tests are created using Postman (and the CLI runner `newman`).
+
+Installation:
+```
+brew install newman
+```
+
+The tests makes use of both pre-request scripts and test scripts in the Postman collection.
+
+The collection runs against a Zendesk instance and an instance of the community-roadmap service. The Zendesk instance must have the Help Center Integration Token endpoint feature enabled (on URL `/hc/api/v2/integration/token`). You should also go to Admin Center to create an API token for your user.
```

Comment:
> instance of the community-roadmap service

What service is this? The one I'm running now?

> The Zendesk instance must have the Help Center Integration Token endpoint feature enabled (on URL `/hc/api/v2/integration/token`).

I'm not familiar with this endpoint, and it looks like it's not documented. Can we link to an article that tells the user how to enable the feature?